pom.xml

```diff
@@ -525,6 +525,42 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
Can you test disabling `attach` configuration in `maven-assembly-plugin` to see if the resulting zips are ignored in maven release?